eng/pipelines/templates/jobs/perf.yml

```diff
@@ -31,11 +34,20 @@ extends:
     - template: /eng/pipelines/templates/variables/globals.yml
     Language: Cpp
     LanguageVersion: 'N/A'
-    InstallLanguageSteps: ${{ parameters.InstallLanguageSteps }}
+    InstallLanguageSteps:
+      - pwsh: |
+          Write-Host "##vso[task.setvariable variable=VCPKG_BINARY_SOURCES_SECRET;issecret=true;]clear;x-azblob,https://cppvcpkgcache.blob.core.windows.net/public-vcpkg-container,,read"
+        displayName: Set Vcpkg Variables
+      - ${{ if parameters.Profile }}:
```

Comment:
Silly question: When is this set to "true"?

Reply:
when you select it in the pipeline run :) 
it's presented as a check box as one of the options of the run , the idea is you want to run profiling for the perf tests , this is how